samples/app-cache-meetings/nodejs/src/components/configure.jsx

```diff
@@ -0,0 +1,38 @@
+// <copyright file="configure.jsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useEffect } from "react";
+import { Text } from "@fluentui/react-northstar";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+// Configure page.
+const Configure = props => {
+    useEffect(() => {
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.app.notifySuccess();
+
+            microsoftTeams.pages.config.registerOnSaveHandler(function (saveEvent) {
+                microsoftTeams.pages.config.setConfig({
+                    //entityId : Generating a random id so that each tab instance has a unique ID.
+                    entityId: Date.now(),
```

Comment:
Consider this instead - easier to read
entityId = "AppInstance_" + Math.floor(Math.random() * 100 + 1);

Reply:
Comment Resolved:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/109147706/211606992-5ca49eac-bba9-48c1-af97-e60a118dcffd.png">
<img width="255" alt="image" src="https://user-images.githubusercontent.com/109147706/211607720-674529c6-ca53-47b5-b765-037e27e973cb.png">